src/schnetpack/transform/neighborlist.py

```diff
@@ -51,13 +51,16 @@ def __init__(
         self,
         cache_path: str,
         neighbor_list: Transform,
+        nbh_transforms: Optional[List[torch.nn.Module]] = None,
         keep_cache: bool = False,
         cache_workdir: str = None,
     ):
         """
         Args:
             cache_path: Path of caching directory.
             neighbor_list: the neighbor list to use
+            nbh_transforms: post-processing transforms for manipulating the neighbor
```

Comment:
post-processing is confusing here, since we usually use this to refer to transforms after the model.